code/modules/antagonists/heretic/knowledge/blade_lore.dm

```diff
@@ -67,40 +67,45 @@
 /datum/heretic_knowledge/blade_grasp/proc/on_mansus_grasp(mob/living/source, mob/living/target)
 	SIGNAL_HANDLER
 
+	if(!check_behind(source, target))
+		return
+
+	// We're officially behind them, apply effects
+	target.AdjustParalyzed(1.5 SECONDS)
+	target.apply_damage(10, BRUTE, wound_bonus = CANT_WOUND)
+	target.balloon_alert(source, "backstab!")
+	playsound(target, 'sound/items/weapons/guillotine.ogg', 100, TRUE)
+
+///Checks to see if `atom/source` is behind `atom/target`
+/proc/check_behind(atom/source, atom/target)
```

Comment:
i think this should be moved to the helpers directory, perhaps in the game.dm file.